Makefile.Common

```diff
@@ -81,6 +81,11 @@ ALL_SRC := $(shell find $(ALL_PKG_DIRS) -name '*.go' \
                                 -not -path '*/local/*' \
                                 -type f | sort)
 
+ALL_SRC_AND_SHELL := $(shell find $(ALL_PKG_DIRS) -name '*.go' -o -name "*.sh" \
+                                -not -path '*/third_party/*' \
+                                -not -path '*/local/*' \
```

Comment:
I noticed `local/` wasn't originally ignored, but found it's in [`.gitignore`,](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/615aad997ab60a8123d5660d61497fce67f16948/.gitignore#L1) so there's no point in including it in makefile logic.

(I agree with your change here, just pointing out why)